docs/content/querying/filters.md

```diff
@@ -133,6 +147,9 @@ Search filters can be used to filter on partial string matches.
 |dimension|The dimension to perform the search over.|yes|
 |query|A JSON object for the type of search. See below for more information.|yes|
```

Comment:
Kept the notices, added an entry for extraction function to the SearchQueryFilter property table